introduction.md

```diff
@@ -3,4 +3,5 @@ Mijn naam is Ivo en ik doe de mastering github course. Mijn favoriet is JavaScri
 * javascript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?